app/assets/stylesheets/custom.scss

```
@@ -0,0 +1,37 @@
+/* universal */
+
+body {
+            padding-top: 60px;
```

wrong indent
fix the same error in this file
```suggestion
  padding-top: 60px;
```